packages/create-cloudflare/templates/hello-world-durable-object/js/src/index.js

```diff
@@ -0,0 +1,58 @@
+/**
+ * Welcome to Cloudflare Workers! This is your first Durable Objects application.
+ *
+ * - Run `npm run dev` in your terminal to start a development server
+ * - Open a browser tab at http://localhost:8787/ to see your Durable Object in action
+ * - Run `npm run deploy` to publish your application
+ *
+ * Learn more at https://developers.cloudflare.com/durable-objects
+ */
+
+/** A Durable Object's behavior is defined in an exported Javascript class */
+export class MyDurableObject {
+	/**
+	 * The constructor is invoked once upon creation of the Durable Object, i.e. the first call to
+	 * 	`DurableObjectStub::get` for a given identifier
+	 *
+	 * @param {DurableObjectState} state - The interface for interacting with Durable Object state
+	 * @param {Env} env - The interface to reference bindings declared in wrangler.toml
```

Comment:
`Env` will need a typedef somewhere in this file

Reply:
Can you do that with JSDoc comments?